src/coreComponents/constitutive/fluid/multifluid/reactive/ReactiveBrineFluid.cpp

```diff
@@ -122,11 +130,16 @@ void ReactiveBrineFluid< PHASE > ::postProcessInput()
                         GEOS_FMT( "{}: invalid number of values in attribute '{}'", getFullName() ),
                         InputError );
 
-  createPVTModels();
+  if( getParent().getName() == ConstitutiveManager::groupKeyStruct::constitutiveModelsString() )
```

Comment:
We should limit ourselves to make assumptions on the `Group` tree. To avoid that, I propose to "tag" the object as a clone by adding a `ConstitutiveBase::isClone` const attribute (default to `false`, and `true` in `ConstitutiveBase::deliverClone()`) and then calling a `ConstitutiveBase::isClone()` when necessary.